platform-dom/src/main/java/com/softicar/platform/dom/elements/popup/configuration/DomPopupDisplayMode.java

```diff
@@ -14,47 +14,50 @@ public enum DomPopupDisplayMode {
 	 * A modal, dismissable, non-draggable {@link DomPopup} without header.
 	 * Provides a visual clue to its modality.
 	 */
-	DIALOG(false, false, DomPopupModalMode.MODAL_DISMISSABLE, DomPopupChildClosingMode.INTERACTIVE_ALL),
+	DIALOG(false, false, false, DomPopupModalMode.MODAL_DISMISSABLE, DomPopupChildClosingMode.INTERACTIVE_ALL),
```

Comment:
- this is getting out of hand, lets spinoff another PR to fix it :)
- I suggest it like this:
```java
public interface DomPopupDisplayModes {
IDomPopupDisplayMode DIALOG = new DomPopupDisplayMode()//
.setCompact(false)
.setMaxmized(false)
...
```

Reply:
Tought the same. Telescope doing telescope things.